UPGRADE-4.2.md

```diff
@@ -0,0 +1,34 @@
+UPGRADE FROM 4.1 to 4.2
+=======================
+
+Form
+----
+
+ * Deprecated `ChoiceLoaderInterface` implementation in `CountryType`, `LanguageType`, `LocaleType` and `CurrencyType`. Use the "choice_loader" option instead.
```

Comment:
"choice_loader" should be enclosed by backticks too

Reply:
Deprecated the `ChoiceLoaderInterface` implementation [...]

Reply:
Fixed, thank you.